### DISCUSSION TITLE
ESC should close command palette

### DISCUSSION THREAD

**Discussion by teemuij** [community/community#28853] [Other Feature Feedback, Questions, & Ideas]
Command palette is nice but when command palette is open, esc key should close it. I'm using macOS and chrome.

**Comment by Akash1134** [comment]
Hi @teemu47 Thanks for reporting this. Would you mind sharing more details around when exactly it is happening and how long have you been facing this issue? 

When the command palette is active, you can use one of the following keyboard shortcuts to close the command palette:

- Search and navigation mode: Esc or Ctrl+K (Windows and Linux) Command+K (Mac)
- Command mode: Esc or Ctrl+Shift+K (Windows and Linux) Command+Shift+K (Mac)

Please note that if you have customized the command palette keyboard shortcuts in the Accessibility settings, your customized keyboard shortcuts will be used for both opening and closing the command palette. Thanks!

**Reply by linco95** [reply]
@Akash1134 I've just tried the command palette for the first time, and I have this issue as well. Closing it with `cmd+k` again works fine (didn't try it until I read this comment), however escape doesn't do anything. (Applies to both command mode, and search mode)

Using: 
`m1 pro`
`macOS 12.5.1`
`chrome Version 106.0.5249.119`

**Comment by Sti2nd** [comment]
I also have this problem. <kbd>ESC</kbd> doesn't close the palette

**Reply by RomainLanz** [reply]
I can confirm this, but using <kbd>Cmd</kbd> + <kbd>K</kbd> does close it.
